### PR TITLE
Please reset all transforms when sidebar closes...

### PR DESCRIPTION
**Pull request by floq-design**
Please reset all transforms when sidebar closes so fixed position child elements of panel work!
See <http://stackoverflow.com/questions/15194313/webkit-css-transform3d-position-fixed-issue>

### CODE REVIEW

File: dist/slideout.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ Slideout.prototype.close = function() {
   this._opened = false;
   setTimeout(function() {
     html.className = html.className.replace(/ slideout-open/, '');
-    self.panel.style.transition = self.panel.style['-webkit-transition'] = '';
+    self.panel.style.transition = self.panel.style[ '-webkit-transition' ] = self.panel.style[ '-webkit-transform' ] = self.panel.style[ 'transform' ] = '';
```
Comment (pazguille) [inline dist/slideout.js]: You should use the `prefix` variable.

```
self.panel.style[prefix + 'transform'] = self.panel.style.transform
```

**Comment by floq-design** [comment]
I've re-submitted a pull request with use of prefix variable.
Missed that one.
Cheers

**Comment by pazguille** [comment]
Thanks! 

Could you use dot notation? 

```js
self.panel.style.transform
```
instead of
```js
self.panel.style['transform']
```

**Comment by pazguille** [comment]
Hi! You should read the following guide before sending pull requests: https://github.com/Mango/slideout/blob/master/CONTRIBUTING.md#modifying-the-code

**Comment by pazguille** [comment]
#54

**Pull request closed without merging**